app/Config/acl.php

```diff
@@ -115,31 +116,31 @@
  * to the roles you defined in the roles configuration.
  */
 $config['map'] = array(
-	'User' => 'User/username',
-	'Role' => 'User/group_id',
+    'User' => 'User/username',
+    'Role' => 'User/group_id',
```

Comment:
__[staging][code_sniffer]__
* Array key not aligned correctly; expected 18 spaces but found 4